modules/swagger-codegen/src/main/resources/typescript-angular/api.service.mustache

```diff
@@ -274,7 +274,8 @@ export class {{classname}} {
             body: {{paramName}} == null ? '' : JSON.stringify({{paramName}}), // https://github.com/angular/angular/issues/10612
 {{/bodyParam}}
 {{#hasFormParams}}
-            body: formParams.toString(),
+            // https://github.com/swagger-api/swagger-codegen/issues/6457
```

Comment:
I'm not sure about that reference. Sometimes there are references to unsolved issues on Angular-side that need to be addressed at some point, but in this case it's just a proper implementation we haven't had before so I don't think that reference is really useful or required. If someone looks up the changes, they will find this PR anyways.

What do you think @TiFu @taxpon @kenisteward ?

Reply:
@sebastianhaas yes, you are right, with git blame one can easily find the last change of that line, so I will remove the comment